tasks/request_files/README.md

```diff
@@ -28,304 +26,4 @@ ex> (podr) λ aws s3api head-object --bucket podaac-sndbx-cumulus-glacier --key
 ```
```

Comment:
May want to add a sentence and link to the schemas like we do in the other Lambdas